src/renderers/WebGLRenderer.js

```diff
@@ -2162,20 +2162,16 @@ class WebGLRenderer {
 			const renderTargetProperties = properties.get( renderTarget );
 			renderTargetProperties.__hasExternalTextures = true;
 
-			if ( renderTargetProperties.__hasExternalTextures ) {
```

Comment:
Right, this check is redundant. 

Strange, I wonder why DeepScan does not detect this line as superfluous 🤔 .

Reply:
Probably because `renderTargetProperties.__hasExternalTextures` is not variable, it is property -- so _theoretically_ it is possible that it is a getter/setter which could produce unexpected results.